src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -318,6 +319,14 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 		}
 	}
 
+	private getCurrentLinkAddress(): string {
+		if (document.getSelection().anchorNode.parentNode['protocol'] === 'file:') {
+			return document.getSelection().anchorNode.parentNode['pathname'] || '';
```

Comment:
It looked like in your example your path started with a / - e.g. `/C:/my/path/notebook.ipynb`. Was this expected?

Reply:
Yes this is the absolute path, similar to that of what I copied into the link originally.

Reply:
I assume this is a windows machine - in which case /C:/my/path wouldn't be a valid path I'd expect. You're saying you entered in the link as `/C:/my/path`? Does that link actually work? Regardless - if you did could you show a gif of updating a link without the / prefix?

Reply:
It is a valid path, as it is the absolute path that can be used for links to a notebook. They get converted to a relative path in markdown preview, so the link does open correctly and is formatted as such in Markdown.
`[WYSIWYG File Test](.\WYSIWYG-test.ipynb)

`![fileLinkWYSIWYG](https://user-images.githubusercontent.com/23587151/113682630-34bd5f80-9678-11eb-9eb7-b3981104ae3e.gif)

Reply:
@VasuBhog - if a link has been resolved to relative link, can you please confirm that updating the link (with the copy path / full path) would still work OK? i.e. the second relative link translation will still happen.

Reply:
@MsSQLGirl, yes if a user enters a new path via the (Copy Path / Full Path) it will be resolved to relative and work as expected.